src/core/util/util.js

```diff
@@ -218,12 +221,18 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
 
       // Creates a virtual scrolling mask to absorb touchmove, keyboard, scrollbar clicking, and wheel events
       function disableElementScroll(element) {
-        element = angular.element(element || body)[0];
-        var scrollMask = angular.element(
-          '<div class="md-scroll-mask">' +
-          '  <div class="md-scroll-mask-bar"></div>' +
-          '</div>');
-        element.appendChild(scrollMask[0]);
+        element = angular.element(element || body);
+        var scrollMask;
+        if (options && options.disableScrollMask) {
+          scrollMask = element;
+        } else {
+          element = element[0];
```

Comment:
@crisbeto I did not change that functionality from the original.

However, I understand that `angular.element` creates a JQLite object of the element and that `[0]` gets the HTML node that can be appended / prepended. Whether or not `element.append` of a JQLite object will work is something that I did not introduce to this functionality.